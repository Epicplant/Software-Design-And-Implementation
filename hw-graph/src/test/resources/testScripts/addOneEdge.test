# Tests adding a singular edge between 2 nodes

# Creates a new graph
CreateGraph newGraph

# Creates the edges starting node
AddNode newGraph n1

# Creates the edges end node
AddNode newGraph n2

# Creates an edge between the starting and end node with weight 0
AddEdge newGraph n1 n2 0

# Lists all of the children of this node
ListChildren newGraph n1