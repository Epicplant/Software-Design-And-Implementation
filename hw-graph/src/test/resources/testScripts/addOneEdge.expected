# Tests adding a singular edge between 2 nodes

# Creates a new graph
created graph newGraph

# Creates the edges starting node
added node n1 to newGraph

# Creates the edges end node
added node n2 to newGraph

# Creates an edge between the starting and end node with weight 0
added edge 0 from n1 to n2 in newGraph

# Lists all of the children of this node
the children of n1 in newGraph are: n2(0)